pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -173,6 +178,9 @@ let
   ++ optional stdenv.hostPlatform.isLinux "ac_cv_func_lchmod=no"
   ++ optional static "LDFLAGS=-static";
 
+  # At this step because it patches the Makefile.
+  preBuild = ''patch -p1 < ${./profile-task.patch}'';
```

Comment:
add the patch to `patches`

Reply:
Sure. That won't work with this patch as written, because it applies to the Makefile and that's generated -- but I now realize I could just patch `Makefile.pre.in` instead. Will do.